src/Fantomas/Trivia.fs

```diff
@@ -282,7 +282,17 @@ let private addTriviaToTriviaNode
 
     | { Item = Keyword({ Content = keyword} as kw); Range = range } when (keyword = "override" || keyword = "default" || keyword = "member" || keyword = "abstract") ->
         findMemberDefnMemberNodeOnLine triviaNodes range.StartLine
-        |> updateTriviaNode (fun tn -> tn.ContentItself <- Some (Keyword(kw))) triviaNodes
+        |> updateTriviaNode (fun tn ->
+            match tn.Type, tn.ContentItself with
+            | MainNode("SynMemberSig.Member"), Some( Keyword ({ Content = existingKeyword })) when (existingKeyword = "abstract" && keyword = "member") ->
```

Comment:
nit: you can remove some unneeded parenthesis here (so that the line is not so long):
```
            | MainNode "SynMemberSig.Member", Some( Keyword { Content = existingKeyword }) when (existingKeyword = "abstract" && keyword = "member") ->
```
Maybe can also remove the parens after `when`? not sure